examples/imagenet/train_imagenet_multi.py

```diff
@@ -68,6 +68,22 @@ def __call__(self, in_data):
         return img, label
 
 
+class FixedBatchDataset(chainer.dataset.DatasetMixin):
+    # Make the dataset size multiple of the batch-size by augmentation
+
+    def __init__(self, dataset, batchsize):
+        self.dataset = dataset
+        self.batchsize = batchsize
+        d = len(self.dataset)
+        self._len = ((d + batchsize - 1) // batchsize) * batchsize
+
+    def __len__(self):
+        return self._len
+
+    def get_example(self, idx):
+        return self.dataset[idx % len(self.dataset)]
```

Comment:
I'm a bit worried about the bias from this. If I understand correctly, this oversample first data twice as many times as other data? Not sure if this is a great idea, but how about returning a data in random index when idx overflows?

Reply:
I noticed that we can specify an "ignore label" for the over-sampled data for the classification problem. This will cause (almost?) no bias in both training and validation.